### PR TITLE
fix: issues-1972:problems caused by too long name

### PR DESCRIPTION
**Pull request by qqqzhch**
### Describe the changes you have made in this PR

- 1  add   breakpoint in tailwind.config.js
-  2 add  a small width size to AccountMenu  in pop view

### Link this PR to an issue [optional]

- #1972 

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

WX20230115-110350

### How has this been tested?

Use a long mailbox name to test

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ module.exports = {
   content: ["./static/views/**/*.html", "./src/app/**/*.{js,ts,jsx,tsx}"],
   plugins: [require("@tailwindcss/forms"), require("@tailwindcss/line-clamp")],
   theme: {
+    screens: {
```
Comment (reneaaron) [inline tailwind.config.js:30]: I don't think we need a new breakpoint in order to limit the width of a component.

**Review by reneaaron — COMMENTED** [review]
Thanks for this PR, however I think there is already an upcoming improvement with #1955 that should be merged soon. 🤞

I don't think we should introduce another breakpoint just do do the layout of one particular use case. (also with *very* long account names this approach wouldn't work on higher resolutions as well) 

I suggest we revisit this PR after #1955 has been merged.

File: tailwind.config.js (hunk 1 above)
Comment (qqqzhch) [reply tailwind.config.js:30]: OK, I will look at the relevant code of # 1955，to see the effect of the solution

Comment (qqqzhch) [reply tailwind.config.js:30]: WX20230116-103840
Just like mobile adaptation, pop view is more like a small-sized mobile page, and the current breakpoint cannot distinguish whether it is running on a smaller-sized page

@rolznz Do you have any ideas, I think this kind of small size problem will be quite common in the future

Comment (rolznz) [reply tailwind.config.js:30]: @reneaaron @qqqzhch I made a new PR here https://github.com/getAlby/lightning-browser-extension/pull/1982 which doesn't introduce a breakpoint. I am not really happy with the solution (it feels a bit hacky), but it is isolated to the account menu.

**Comment by reneaaron** [comment]
Closing this PR in favor of #1982 which should be merged soon. Thanks for your initiative on this problem @qqqzhch!

**Pull request closed without merging**